src/include/btmem.h

```diff
@@ -6,6 +6,8 @@
  * See the file LICENSE for redistribution information.
  */
 
+#define	WT_RECNO_OOB	0		/* Illegal record number */
```

Comment:
I've done a pass. I'm sure it's not complete, but I agree it's better than it was, and we can fix the remaining issues over time.